src/main/resources/i18n/messages_cs.properties

```diff
@@ -74,8 +74,8 @@ ranked1v1.rankingOutOf = ze {0,number,\#} hráčů
 ranked1v1.searching = Hledám soupeře...
 ranked1v1.winLossRation = Poměr výher/proher
 ranked1v1.rankingFormat = \#{0,number,\#}
-ranked1v1.notification.title = Dostupná 1v1 Ranked hra
-ranked1v1.notification.message = Klikni zde pro otevření 1v1 záložky
+ranked1v1.availableOpponentNotification.title = Dostupná 1v1 Ranked hra
+ranked1v1.availableOpponentNotification.message = Klikni zde pro otevření 1v1 záložky
```

Comment:
Ah, I was renaming an existing property since `ranked1v1.notification` became a bit ambiguous with the new notifications.
Modifying `src/main/resources/i18n/messages.properties` for the defaults is fine, is that right?

I guess I will create the renamed property in `messages.properties` and see if I can get poeeditor to copy over all translations? From what I've heard one might the right to do that on a per-language basis?